pkg/models/possible_operational_solution.go

```diff
@@ -57,6 +57,7 @@ const (
 	OpSKISP                       OperationalSolutionKey = "ISP"
 	OpSKMIDS                      OperationalSolutionKey = "MIDS"
 	OpSKModelSpace                OperationalSolutionKey = "MODEL_SPACE"
+	OpSKCdac                      OperationalSolutionKey = "CDAC"
```

Comment:
I think I might have misunderstood earlier, does this need to be added to possible operational solution? I'd imagine it would be ok to leave out adding this, even though the type remains.

Reply:
I initially had it, then I removed it and confirmed with you that we would no longer need it here.  Should I put it back?

Reply:
We aren't using it anywhere, except as a type in change history.  But that would never really be used anyway.  There isnt ever a case where we would translate OperationalSolutionKey in the new state of the app

Reply:
Anddddd I just realized I never pushed up the code that removed it again